pkg/config/legacy_converter.go

```diff
@@ -0,0 +1,27 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+package config
+
+import "strings"
+
+// LegacyConfigConverter should only be used in tests
```

Comment:
the comment doesn't seem to reflect that this is actually used in the `legacy` package, and not in tests.

Same comment on the two other function comments below